config/routes.rb

```diff
@@ -18,6 +18,8 @@
 
   resources :collections, only: %i[new create show edit update destroy] do
     member do
+      get :edit_first_draft
```

Comment:
Please make these two methods on their own controller.  We are saying "a first draft collection is so different that it ought to be treated as its own kind of resource".

Reply:
Done